File: C/Numbers/program8.c
// Program to display N times Hello on screen

#include<stdio.h>

void Display(int iNo)
{
    int iCnt = 0;
    if(iNo < 0)             // Filter
    {
        printf("Please enter positive value\n");
        return;
    }

    for( iCnt = 0;  iCnt<iNo;  ++iCnt)
    {
        printf("Hello\n");
    }
}

int main()
{
    int iValue = 0;

    printf("Enter the number\n");
    scanf("%d",&iValue);

    Display(iValue);
    
    return 0;
}

File: Java/Numbers/program8.java
// Program to display 5 times Hello on screen

import java.util.*;

class Number
{
    public void Display(int iNo)
    {
        int iCnt = 0;

        if(iNo < 0)
        {
            System.out.println("Please enter positive value");
            return;
        }

        for(iCnt = 1; iCnt <= iNo; ++iCnt)
        {
            System.out.println("Hello");
        }
    }
}

class program8
{
    public static void main(String b[])
    {
        int iValue = 0;

        Scanner sobj = new Scanner(System.in);
        
        System.out.println("Enter the number");
        iValue = sobj.nextInt();

        Number obj = new Number();

        obj.Display(iValue);
    }
}